node/overseer/Cargo.toml

```diff
@@ -18,7 +18,7 @@ polkadot-primitives = { path = "../../primitives" }
 polkadot-overseer-gen = { path = "./overseer-gen" }
 tracing = "0.1.31"
 lru = "0.7"
-parity-util-mem = { version = ">= 0.10.1", default-features = false }
+parity-util-mem = { version = "0.11.0", default-features = false }
```

Comment:
This change is narrowing the allowed versions from both directions.

Reply:
parity-util-mem doesn't allow duplicates on the link level, so it doesn't change anything